index.html

```diff
@@ -1,172 +1,311 @@
-                <li class="about__link">
-                    <a href="#"><img src="./images/github.svg" alt="my github profile URL"></a>
-                </li>
-                <li class="about__link">
-                    <a href="#"><img src="./images/Linkedin.svg" alt="my Linkedin profile URL"></a>
-                </li>
-                <li class="about__link">
-                    <a href="#"><img src="./images/angellist.svg" alt="my angellist profile URL"></a>
-                </li>
-                <li class="about__link">
-                    <a href="#"><img src="./images/twitter.svg" alt="my twitter profile URL"></a>
-                </li>
-                <li class="about__link">
-                    <a href="#"><img src="./images/medium.svg" alt="my medium profile URL"></a>
-                </li>
-            </ul>
-            <button class="btn btn-primary" type="button">Get my resume</button>
-            <ul class="about__categories">
-                <li class="menu__link">
-                    <a href="#">Languages<img src="./images/IconDown.svg" alt=""></a>
-                    <ul class="about__categories-submenu">
-                        <li>
-                            <a class="submenu__link" href="#"><img src="./images/javascript.svg" alt=""> JavaScript</a>
-                        </li>
-                        <li>
-                            <a class="submenu__link" href="#"><img src="./images/html.svg" alt=""> HTML</a>
-                        </li>
-                        <li>
-                            <a class="submenu__link" href="#"><img src="./images/css.svg" alt=""> CSS</a>
-                        </li>
-                    </ul>
-                </li>
-                <li class="menu__link">
-                    <a href="#">Frameworks<img src="./images/IconRight.svg" alt=""></a>
-                </li>
-                <li class="menu__link">
-                    <a href="#">Skills<img src="./images/IconRight.svg" alt=""></a>
-                </li>
-
-            </ul>
-        </section>
-        
+          </li>
+          <li class="menu__link">
+            <a href="#"
+              >Frameworks<img src="./images/IconRight.svg" alt=""
+            /></a>
+          </li>
+          <li class="menu__link">
+            <a href="#">Skills<img src="./images/IconRight.svg" alt="" /></a>
+          </li>
+        </ul>
+      </section>
     </main>
-</body>
-</html>
\ No newline at end of file
+    <footer class="footer">
+      <fieldset>
+        <form
+          class="form"
+          action="https://formspree.io/f/mgedpkaq"
+          method="POST"
+        >
+          <h3 class="form-title">Contact me</h3>
+          <p class="form-description">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d
+            love to help with it
+          </p>
+          <ul class="form-list">
+            <li>
```

Comment:
- [ ] In my opinion, you can make your form more leaner and easy to read by removing the use of `<ul>` and its `<li>` from here. You can use something like:
```
<form>
   <input name />
   <input email />
   <textarea message />
   <button type="submit" />
</form>
```